pkgs/development/tools/build-managers/buildbot/plugins.nix

```diff
@@ -4,11 +4,11 @@ let
   buildbot-pkg = pythonPackages.buildPythonPackage rec {
     name = "${pname}-${version}";
     pname = "buildbot-pkg";
-    version = "0.9.7";
+    version = "0.9.9.post1";
 
     src = pythonPackages.fetchPypi {
       inherit pname version;
-      sha256 = "0p351r10y42gwgxb2qg7xlsbhmnzdmqp6h4922l0yfii3pzmrdzv";
+      sha256 = "0000000000000000000000000000000000000000000000000000";
```

Comment:
If the lack of sdist is preventing this from going in, then let's just use wheels or tarballs from GitHub.